internal/dumper/dumper.go

```diff
@@ -223,43 +223,28 @@ func (d *Dumper) dumpTable(conn *Connection, database string, table string) erro
 	var where string
 	var selfields []string
 
-	isGenerated, err := d.generatedFields(conn, table)
-	if err != nil {
-		return err
-	}
-
 	fields := make([]string, 0)
 	{
-		cursor, err := conn.StreamFetch(fmt.Sprintf("SELECT * FROM `%s`.`%s` LIMIT 1", database, table))
+		flds, err := d.dumpableFieldNames(conn, table)
```

Comment:
This is the primary change. This new function uses `show fields` to grab all the dumpable field names (any field that isn't generated).